flask_caching/__init__.py

```diff
@@ -71,6 +71,14 @@ def get_arg_default(f, position):
     return arg_def if arg_def != inspect.Parameter.empty else None
 
 
+def get_id_func(obj):
```

Comment:
Why this separate function is needed here? As i see you only use this value within `get_id`.

Reply:
For future reuse and ease readability?

Reply:
I simply canʼt think of a use case when you need the id generating function but not the ID. And this way you add an unnecessary function call to a oneliner with no significant improvement on readability. How about something like this?

```
def get_id(obj):
    id_func = getattr(obj, '__caching_id__', repr)

    return id_func(obj)
```

Reply:
🤷‍♂